src/components/FileContributors.tsx

```diff
@@ -19,60 +16,21 @@ import {
 } from "@chakra-ui/react"
 import { FaGithub } from "react-icons/fa"
 
-import { getLocaleTimestamp } from "../utils/time"
-import { trackCustomEvent } from "../utils/matomo"
-import { Lang } from "../utils/languages"
-
-import { Button, ButtonLink } from "./Buttons"
-import InlineLink from "./Link"
-import Modal from "./Modal"
-import Translation from "./Translation"
-import Text from "./OldText"
-
-export interface Author {
-  name: string
-  email: string
-  avatarUrl: string
-  user: {
-    login: string
-    url: string
-  }
-}
+import { Button, ButtonLink } from "@/components/Buttons"
+import InlineLink from "@/components/Link"
+import Modal from "@/components/Modal"
+import Translation from "@/components/Translation"
+import Text from "@/components/OldText"
 
-export interface Commit {
-  author: Author
-  committedDate: string
-}
+import { getLocaleTimestamp } from "@/lib/utils/time"
+import { trackCustomEvent } from "@/lib/utils/matomo"
+import type { Lang } from "@/lib/types"
+import type { Author } from "@/lib/interfaces"
 
-const COMMIT_HISTORY = gql`
-  query CommitHistory($relativePath: String) {
-    repository(name: "ethereum-org-website", owner: "ethereum") {
-      ref(qualifiedName: "master") {
-        target {
-          ... on Commit {
-            id
-            history(path: $relativePath) {
-              edges {
-                node {
-                  author {
-                    name
-                    email
-                    avatarUrl(size: 100)
-                    user {
-                      login
-                      url
-                    }
-                  }
-                  committedDate
-                }
-              }
-            }
-          }
-        }
-      }
-    }
-  }
-`
+// export interface Commit {
+//   author: Author
+//   committedDate: string
+// }
```

Comment:
✂️ ?

Reply:
Eventually... IIRC this was used more on the Crowdin side so I wanted to leave it for that. I'll add a TODO

Reply:
Actually, it's only used in that file, so I'm just going to move it over there and can handle it further in the Crowdin PR